04_numeronym/satish/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := "[a11y K8s abc]"
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+var tests = []struct {
+	in  []string
+	out []string
+}{
+	{[]string{"internationalization", "Administration", "Africanization", "Appocalypse"}, []string{"i18n", "A12n",
```

Comment:
Love the creativity with the inputs.
Please add unicode, empty string, no string and whitespace tests.